src/expressions.md

```diff
@@ -45,13 +45,12 @@
 An expression may have two roles: it always produces a *value*, and it may have
 *effects* (otherwise known as "side effects"). An expression *evaluates to* a
 value, and has effects during *evaluation*. Many expressions contain
-sub-expressions (operands). The meaning of each kind of expression dictates
```

Comment:
This is a nit, and not a semantic issue with the description, but: I have a moderate preference that we continue to use "sub-expressions" in general, and only use "operands" for "sub-expressions of an operator".

Reply:
Withdrawing this comment; the only case that really bugs me is the idea that in `f(a, b)`, `f`, `a`, and `b` would all be considered "operands" of the function call expression, and I can live with that terminology in the interests of precision.

Reply:
FWIW, I didn't like the word beforehand, but after this and the tuple PR, I've found it works well. The only alternative I can think of would be to state that when we talk about sub-expressions, we only mean the top-most level, not the subexpressions of the subexpressions. And requiring global context seems like a bad idea.